facia-purger/cloudformation/cloudformation.json

```diff
@@ -0,0 +1,121 @@
+  "Description": "Facia purger lambda",
+  "Parameters": {
+    "LambdaName": {
+      "Description": "Lambda name",
+      "Type": "String",
+      "Default": "facia-purger"
+    },
+    "Stage": {
+      "Description": "Stage name",
+      "Type": "String",
+      "AllowedValues": [
+        "CODE",
+        "PROD"
+      ],
+      "Default": "CODE"
+    },
+    "DeployBucket": {
+      "Description": "Bucket where RiffRaff uploads artifacts on deploy",
+      "Type": "String"
+    }
+  },
+  "Resources": {
+    "ExecutionRole": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "AssumeRolePolicyDocument": {
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Principal": {
+                "Service": [
+                  "lambda.amazonaws.com"
+                ]
+              },
+              "Action": "sts:AssumeRole"
+            }
+          ]
+        },
+        "Path": "/",
+        "Policies": [
+          {
+            "PolicyName": "logs",
+            "PolicyDocument": {
+              "Statement": {
+                "Effect": "Allow",
+                "Action": [
+                  "logs:CreateLogGroup",
+                  "logs:CreateLogStream",
+                  "logs:PutLogEvents"
+                ],
+                "Resource": "arn:aws:logs:*:*:*"
+              }
+            }
+          },
+          {
+            "PolicyName": "lambda",
+            "PolicyDocument": {
+              "Statement": {
+                "Effect": "Allow",
+                "Action": [
+                  "lambda:InvokeFunction"
+                ],
+                "Resource": "*"
+              }
+            }
+          },
+          {
+            "PolicyName": "s3",
+            "PolicyDocument": {
+              "Statement": {
+                "Effect": "Allow",
+                "Action": [
+                  "s3:GetObject"
+                ],
+                "Resource": "arn:aws:s3:::aws-frontend-store/*"
+              }
+            }
+          }
+        ]
+      }
+    },
+    "Lambda": {
+      "Type": "AWS::Lambda::Function",
+      "Properties": {
+        "Code": {
+          "S3Bucket": {
+            "Ref": "DeployBucket"
+          },
+          "S3Key": {
+            "Fn::Join": [
+              "/",
+              [
+                { "Ref": "Stage" },
+                "lambda",
+                {
+                  "Fn::Join": [
+                    "",
+                    [ { "Ref": "LambdaName" }, ".jar"]
+                  ]
+                }
+              ]
+            ]
+          }
+        },
+        "Description": "Purge fastly after a facia change event from an S3 bucket",
+        "Handler": "com.gu.purge.facia.Lambda",
+        "MemorySize": 256,
+        "Role": {
+          "Fn::GetAtt": [
+            "ExecutionRole",
+            "Arn"
+          ]
+        },
+        "Runtime": "java8",
+        "Timeout": 20
+      }
+    }
+  }
+}
```

Comment:
should we (not needed in this PR) have an alarm if the number of purges drops below some value? or if there are more than x failed purges?  just in case someone breaks the fastly key, or changes the location of the pressed json file within the bucket,